Source/DafnyCore/Verifier/Translator.TrStatement.cs

```diff
@@ -218,10 +218,9 @@ public partial class Translator {
           TrStmt(resolved[0], builder, locals, etran);
         } else {
           AddComment(builder, s, "update statement");
-          var lhss = new List<Expression>();
-          foreach (var lhs in s.Lhss) {
```

Comment:
`s.Lhss` and `s.Rhss` should no longer be accessed after resolution, since they are the unresolved children of `UpdateStmt`. Instead, `s.ResolvedStatements` should be used.